docs/resources/Sticker.md

```diff
@@ -114,15 +114,15 @@ Returns the list of sticker packs available to Nitro subscribers.
 
 ## List Guild Stickers % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/stickers
 
-Returns an array of [sticker](#DOCS_RESOURCES_STICKER/sticker-object) objects for the given guild. Includes `user` fields if the bot has the `MANAGE_GUILD_EXPRESSIONS` permission.
+Returns an array of [sticker](#DOCS_RESOURCES_STICKER/sticker-object) objects for the given guild. Includes `user` fields if the bot has the `CREATE_GUILD_EXPRESSIONS` or `MANAGE_GUILD_EXPRESSIONS` permission.
```

Comment:
With `CREATE_GUILD_EXPRESSIONS` permission only, does it still include the `user` field on all sticker objects or does it follow the same rules as retrieving a single sticker object for each of the list's items?

Reply:
see the note in the pr description - all endpoints except getting a single emoji always return user objects with `CREATE_GUILD_EXPRESSIONS`

Reply:
Ah, I get now, thanks!